styles.css

```diff
@@ -33,23 +33,64 @@ body {
 }
 
 .hamburguer {
+  position: relative;
   margin-right: 36px;
   height: 12px;
   width: 18px;
+  z-index: 10;
 }
 
 .bar {
   display: block;
-  background-color: #6070FF;
+  background-color: #6070ff;
   width: 20px;
   height: 2px;
   margin: 2px auto;
   -webkit-transition: all 0.3s ease-in-out;
   transition: all 0.3s ease-in-out;
 }
 
+.hamburguer.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburguer.active .bar:nth-child(1) {
+  transform: translateY(4px) rotate(45deg);
+  background-color: #fff;
+}
+
+.hamburguer.active .bar:nth-child(3) {
+  transform: translateY(-4px) rotate(-45deg);
+  background-color: #fff;
+}
+
 .nav_menu {
-  display: none;
+  position: fixed;
+  z-index: 0;
+  left: 100%;
+  top: 0;
+  gap: 10%;
+  flex-direction: column;
+  padding-top: 70px;
+  background-color: #6070ff;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(5px);
+  width: 100%;
+  height: 100%;
+  text-align: left;
+  transition: 0.3s;
+}
+
+.nav_li_item {
+  text-decoration: none;
+  font-weight: 500;
```

Comment:
Great job with the font size 👍  But let's stick to the Figma design guideline and fix the font weight. It needs to be 600.
<img width="542" alt="Screenshot 2023-04-10 at 11 43 03 PM" src="https://user-images.githubusercontent.com/25887253/230995692-b904b08f-0eef-47a6-b5f9-6ed94c244633.png">
<img width="779" alt="Screenshot 2023-04-10 at 11 44 32 PM" src="https://user-images.githubusercontent.com/25887253/230995702-eb43712a-164b-4f1b-a704-9f39ac973f65.png">